linode/manager
# fix: [M3-8572] - Scrollbar showing briefly on Splash Screen

## Description 📝

- Fixes bug where a scrollbar would briefly show on the splash screen due to missing `sr-only` css class 📄 

## Preview 📷

| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/user-attachments/assets/98b079b5-f789-4f59-b69c-0f09a8af03e5" /> | <video src="https://github.com/user-attachments/assets/c9c9a809-3044-4a6f-a9dc-4fd56385c02d" /> |

## How to test 🧪

- Verify you don't see a scrollbar on the initial load of Cloud Manager
- If you want the splash screen to stay open for testing, go to line `30` of `App.tsx` and force the splash screen to stay open

## As an Author I have considered 🤔
